src/js/components/RadioButton/RadioButton.js

```diff
@@ -34,11 +64,34 @@ class RadioButton extends Component {
             })}
             theme={theme}
           />
-          <StyledRadioButtonButton theme={theme}>
-            <svg viewBox='0 0 24 24' preserveAspectRatio='xMidYMid meet'>
-              <circle cx={12} cy={12} r={6} />
-            </svg>
-          </StyledRadioButtonButton>
+          <StyledRadioButtonBox
+            theme={theme}
+            focus={focus}
+            as={Box}
+            align='center'
+            justify='center'
+            width={theme.radioButton.size}
+            height={theme.radioButton.size}
+            border={{
```

Comment:
Can we just set `border={theme.radioButton.border}`?

Reply:
yes! we can actually pass just the color name in the custom theme and rely on normalizeColor to parse that properly